### ISSUE TITLE
Detect which view is the active view (i.e. view === "canvas")

### ISSUE THREAD

**Issue by jonathanherdt** [storybookjs/storybook#8474]
**Is your feature request related to a problem? Please describe.**
I'm trying to set up a part of a story that should only show up when in Docs view because it does not make sense in other views.

**Describe the solution you'd like**
Inside my `.stories` file, I would like to be able to determine whether the current view is the docs view. Optimally, I would like something like this:

```
storiesOf("MyComponent", module)
	.add("test", (context) => {
		if (context.view === "canvas") {
			// something awesome
		}
	});
```

**Describe alternatives you've considered**
In my case, I have the option of foregoing the Canvas view altogether and just write an MDX file

**Are you able to assist bring the feature to reality?**
if I get some setup help, I'll gladly help :)

**Additional context**
// none

**Comment by shilman** [comment]
@yannbf can this be closed with a link to the documentation?

**Comment by yannbf** [comment]
Hey @shilman not yet because the functionality and documentation is [here](https://github.com/storybookjs/storybook/pull/12368/files#diff-06103ba327f3eb6233073a9c8a9afb89R15) as part of an open PR, but as soon as it's merged then yes 💃

**Comment by shilman** [comment]
Fixed in https://github.com/storybookjs/storybook/pull/12566 and released in 6.1.0-alpha.15. Closing!
